lib/rubocop/cop/layout/space_inside_reference_brackets.rb

```diff
@@ -110,8 +110,14 @@ def bracket_method?(node)
           BRACKET_METHODS.include?(method)
         end
 
-        def left_ref_bracket(tokens)
-          tokens.reverse.find(&:left_ref_bracket?)
+        def left_ref_bracket(node, tokens)
+          _, method = *node
```

Comment:
Method send nodes are decorated with the `SendNode` extension, which allows you to do things like:

```ruby
send_node.method?(:[]=)
```

Reply:
Are you saying I could use `node.method?(:[]=)` instead of `method == :[]=` on line 116?

Reply:
Yes. And skip the destructuring on L:114. 🙂

Reply:
Awesome, I'll give that a go.